packages/icons/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/icons",
-  "version": "1.111.0",
+  "version": "1.111.1",
```

If this is adding something and not _only_ fixing a bug, it should be a minor release, not a point release
```suggestion
  "version": "1.112.0",
```

---

done